net/wireguard/src/www/widgets/widgets/wireguard.widget.php

```diff
@@ -30,6 +30,32 @@
 require_once("guiconfig.inc");
 require_once("widgets/include/wireguard.inc");
 
+function getPubkeyNames()
+{
+    $mdlclients = new OPNsense\Wireguard\Client();
+    $search = $mdlclients->getNodes();
+
+    $ret = array();
+    if(isset($search['clients']['client']) && is_array($search['clients']['client']))
+    {
+        foreach($search['clients']['client'] as $client)
+        {
+            $ret[$client['pubkey']] = $client['name'];
+        }
+    }
+    return $ret;
+}
+
+function getInterfaceNames()
```

Comment:
this seems too hacky, i cannot describe it differently

Reply:
exists already a other function?